scripts/linux-ubuntu/pbuilderrc

```diff
@@ -1,6 +1,5 @@
-QT4_PACKAGES="libqt4-dev libqt4-opengl libqt4-svg libqt4-xmlpatterns libqt4-opengl-dev libqt4-sql libqt4-sql-sqlite qtkeychain-dev"
```

Comment:
If you remove the Qt 4 packages from the pbuilder chroots then we won't be able to build 2.1 anymore. If we never need to build it again then that's fine.

Reply:
We don't need to build 2.1 anymore.

Reply:
I was assuming that these are obsolete on this 2.3 branch. But if the file is then used to define a single builder that is used for multiple Mixxx versions this is misleading.

Reply:
These scripts are for creating / updating an Ubuntu build VM and the pbuilder chroot environments, so unfortunately there is no distinction.